insights/parsers/ironic_conf.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+IronicConf - file ``/etc/ironic/ironic.conf``
+=============================================
+This class provides parsing for the file ``/etc/ironic/ironic.conf``.
+
+Sample input data is in the format::
+    [DEFAULT]
+    auth_strategy=keystone
+    default_resource_class=baremetal
+    enabled_hardware_types=idrac,ilo,ipmi,redfish
+    enabled_bios_interfaces=no-bios
+    enabled_boot_interfaces=ilo-pxe,pxe
+    enabled_console_interfaces=ipmitool-socat,ilo,no-console
+    force_raw_images = true
+
+    [agent]
+    deploy_logs_collect=always
+    deploy_logs_storage_backend=local
+    deploy_logs_local_path=/var/log/ironic/deploy/
+
+    [cinder]
+    auth_url=http://1.1.1.1:5000
+    project_domain_name=Default
+    project_name=service
+    user_domain_name=Default
+
+Examples:
+    >>> conf = shared[IronicConf]
+    >>> conf.sections()
+    ['DEFAULT', 'agent', 'cinder']
+    >>> 'agent' in conf
+    True
+    >>> conf.has_option('DEFAULT', 'auth_strategy')
+    True
+    >>> conf.get("DEFAULT", "enabled_hardware_types")
+    "idrac,ilo,ipmi,redfish"
+    >>> conf.get("DEFAULT", "default_resource_class")
+    "baremetal"
+    >>> conf.getboolean("DEFAULT", "force_raw_images")
+    True
+
+"""
+
+from .. import IniConfigFile, parser, add_filter
+from insights.specs import Specs
+
+add_filter(Specs.ironic_conf, ["["])
+
+
+@parser(Specs.ironic_conf)
+class IronicConf(IniConfigFile):    
```

Comment:
You will see a flake8 error on this line for trailing blank space after the `:`.  You can run `flake8` from the root of your repo to see ths.